qat_policy_cifar100.yaml

```diff
@@ -1,5 +1,5 @@
-start_epoch: 80
```

Comment:
There is another yaml called "qat_policy_late_cifar.yaml" that is used to train cifar100-qat model. They resemble each other, both starts QAT late. Should we integrate them?